internal/core/src/query/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ set(MILVUS_QUERY_SRCS
         SearchOnIndex.cpp
         SearchBruteForce.cpp
         SubSearchResult.cpp
-        GroupByOperator.cpp
+        groupby/SearchGroupByOperator.cpp
         PlanProto.cpp
```

Comment:
As we will support query_groupby in the near future, so put SearchGroupBy code into groupby directory and diff searchGroupBy and queryGroupBy